src/Admin/Menu/MainMenuBuilder.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Added in error?

Reply:
Yes, deleted here https://github.com/netgen/media-site/pull/194/commits/0c730f88e1cced16467dbb166e16624ac37fccc8